03/app.js

```diff
@@ -1,3 +1,12 @@
 export default function randomNumber(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
```

Comment:
Tutaj zamieniasz to na liczbę.
Przed takim zapisem powinien być waurnek tj.
```
if(typeof min !== 'number') {
        throw new Error('not number');
 } 
```

Też zwróć uwagę, że zapis: `min = Math.ceil(min);` jest zły ponieważ modyfikujesz parametr.
Tak nie powinno się robić. Lepiej zrobić nową zmienną: `const minInt = Math.ceil(min);`